tests/rt_35d.conf

```diff
@@ -2,12 +2,12 @@
 # CPLD Benchmark 35d tests                                                                                                                                                    #
 ###############################################################################################################################################################################
 
-COMPILE | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8                                                                 | + hera.intel                | fv3 |  |
-RUN     | cpld_bmark_p8_35d                                                                                               |                             | fv3 |  | 2012010100
-RUN     | cpld_bmark_p8_35d                                                                                               |                             | fv3 |  | 2012040100
-RUN     | cpld_bmark_p8_35d                                                                                               |                             | fv3 |  | 2012070100
-RUN     | cpld_bmark_p8_35d                                                                                               |                             | fv3 |  | 2012100100
-RUN     | cpld_bmark_p8_35d                                                                                               |                             | fv3 |  | 2013010100
-RUN     | cpld_bmark_p8_35d                                                                                               |                             | fv3 |  | 2013040100
-RUN     | cpld_bmark_p8_35d                                                                                               |                             | fv3 |  | 2013070100
-RUN     | cpld_bmark_p8_35d                                                                                               |                             | fv3 |  | 2013100100
+COMPILE | 1 | intel | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | + hera.intel | fv3 |  |
```

Comment:
What should we do (if anything) about the fv3 in the compile line? Does this just name the executable? I really don't know what it does @DusanJovic-NOAA

Reply:
I agree, im still not sure what it does, i think we used it long ago so we didn't have to redo compile? We can address this in a later PR.

Reply:
That sounds fine. There is a catch-all issue #1756 it could be part of.

Reply:
> What should we do (if anything) about the fv3 in the compile line? Does this just name the executable? I really don't know what it does @DusanJovic-NOAA

No, it does not name the executable. It has the same meaning as the string in the 4th column in RUN entries. If rt.sh is running in create baselines mode, and if the 4th column is (currently) 'fv3' it will run the COMPILE step. That's all, nothing to do with the executable name.

The reason this exists is when we implemented rt.sh originally NEMS had multiple models, gfs, nmm, fim etc. These were completely different models, so very often code changes will be local to one particular model and if new baselines are required it made sense to create them only for that model only. Back then the '-c' flag had an additional option which would specify the model. For example:

```
./rt.sh -c nmm
```
would create new baselines only for the NMM model and copy current baselines for other two models.

Now that we have only one model that option is redundant and we run all COMPILE steps all the time, and that's why each COMPILE line has that 'fv3' in the 4th column. We can keep it for consistency with RUN entries, but the 'fv3' string can be changed. Or we can simply remove it.